core/src/Streamly/Internal/Data/Parser/Chunked.hs

```diff
@@ -7,6 +7,18 @@
 -- Maintainer  : streamly@composewell.com
 -- Stability   : pre-release
 -- Portability : GHC
+--
+-- Differences from an item parser:
```

Comment:
`element parser`?
`item parser` sounds weird.

Reply:
This is internal, element/item sound the same to me, can make it element.